frontend/src/main.js

```diff
@@ -30,6 +30,11 @@ loadAllRules(i18n)
 
 addNavigationGuards(router, store, apolloProvider.defaultClient)
 
+if (!store) {
+  // Please supply a support page
+  window.location.assign('https://gradido.net/')
```

Comment:
Like @ulfgebhardt said yesterday, would be prettier if we could have a page in our application